test/unit/user_test.rb

```diff
@@ -19,16 +21,11 @@ class UserTest < ActiveSupport::TestCase
 
   setup do
     ActionMailer::Base.deliveries = []
-    # This is needed, database will not save fractions of seconds
-    # so something like:
-    # Time.now.to_f #=> 1512012308.293877
-    # will be saved in DB as 1512012308
-    Timecop.freeze(Time.parse('2017-11-23 03:25:08 UTC +00:00'))
   end
 
-  def teardown
-    Timecop.return
-  end
+  # def teardown
+  #   Timecop.return
+  # end
```

Comment:
why comment?

Reply:
There are probably many things left undone so let's split this into smaller PRs to make it easier for everyone.